Gemfile.lock

```diff
@@ -223,9 +223,9 @@ DEPENDENCIES
   minitest (~> 5.14)
   rake (~> 13.0)
   rubocop (~> 0.90.0)
+  rubyzip (~> 2.3.0)
   simctl (~> 1.6)
   xcodeproj (~> 1.18)
-  zip (~> 2.0.2)
 
 BUNDLED WITH
-   2.1.2
```

Comment:
Would you mind updating all the [GitHub workflows](https://github.com/tuist/tuist/blob/master/.github/workflows/tuist.yml#L83) to install the same Bundler version as the one specified here (2.1.4)

Reply:
please verify if I did it correctly